winui/Ribbon/Ribbon-QAT.md

```diff
@@ -121,6 +122,22 @@ The Ribbon also supports adding items to the QAT Menu items. To add items to the
 
 ![WinUI Ribbon QAT menu items](Ribbon-QAT-images/winui-ribbon-qat-menu-items.png)
 
+## Adding items using QAT Customize dialog window
+
+To open QAT customize dialog window, select `MoreCommands` option from the ContextMenu of the QAT. This dialog window showcases all available commands within the Ribbon control.
```

Comment:
To open the QAT customize dialog window, select the `MoreCommands` option from the ContextMenu of the QAT. This dialog window showcases all available commands within the Ribbon control.

Reply:
modified